pkg/combustion/kubernetes.go

```diff
@@ -57,6 +58,25 @@ func configureKubernetes(ctx *image.Context) ([]string, error) {
 		return nil, fmt.Errorf("configuring kubernetes components: %w", err)
 	}
 
+	manifestURLs := ctx.ImageDefinition.Kubernetes.Manifests.URLs
```

Comment:
All of this belongs to `configureManifest` instead. Here we should only be doing:

```
if err = configureManifests(...); err != nil {
	log.AuditComponentFailed(k8sComponentName)
	return nil, fmt.Errorf("configuring kubernetes manifests: %w", err)
}
```

Reply:
I was thinking about that, will refactor